browse/routes/ui.py

```diff
@@ -323,6 +323,14 @@ def archive(archive: str):  # type: ignore
     return response, code, headers
 
 
+@blueprint.route('archive/<archive>/<junk>', strict_slashes=False)
+def archive_with_extra(archive: str, junk: str):  # type: ignore
+    """Archive page with extra, 301 redirect to just the archive.
```

Comment:
Also minor nit, but per [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings):

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. The summary line may be on the same line as the opening quotes or on the next line. 

Regarding the last point, in most cases throughout NG code we seem to be putting the summary line on the line following the opening quotes.

Reply:
Fixed!